js/components/InfoForm/index.js

```diff
@@ -100,12 +100,13 @@ class Form extends Gandalf {
     if (!data) return;
     data = {
       ...data,
-      date: this.state.date.toLocaleDateString([], {year: 'numeric', month: 'short', day: 'numeric'}),
-      arriveTime: this.state.arriveTime.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'}),
-      servingTime: this.state.servingTime.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'})
+      date: this.state.date.toLocaleDateString([], { year: 'numeric', month: 'short', day: 'numeric' }),
+      arriveTime: this.state.arriveTime.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' }),
+      servingTime: this.state.servingTime.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' })
     };
     // Submit to REDUX
     console.log('goin\' to REDUX', data);
+    console.log('this.props.navigateTo');
```

Comment:
Do you mean for this to be a variable or a string? (Or should it even be there?)